borgmatic/commands/hook.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+
+
+def exec_cmd(config):
+    if config and config.get('enable_hook', None) is True:
```

Comment:
Cool, sounds good.

Side note: You bring up an interesting point that the `generate-borgmatic-config` file isn't really usable as-is. I wonder if it would be helpful to have a way to generate a more minimal file that is closer to usable out of the box.